avaframe/com2AB/com2AB.py

```diff
@@ -245,6 +246,33 @@ def findSplitPoint(AvaProfile, splitPoint):
     return SplitPoint, splitPoint
 
 
+
+def readABinputs(cfgINPATH):
+    cfgAva = cfgINPATH['AvaLancheRoot']
+    cfgPath = {}
+
+    ProfileLayer = glob.glob(cfgAva + '/Inputs/LINES/*.shp')
+    cfgPath['ProfileLayer'] = ''.join(ProfileLayer)
+
+    DGMSource = glob.glob(cfgAva + '/Inputs/*.asc')
+    cfgPath['DGMSource'] = ''.join(DGMSource)
+
+    SplitPointSource = glob.glob(cfgAva + '/Inputs/POINTS/*.shp')
+    cfgPath['SplitPointSource'] = ''.join(SplitPointSource)
+
+    saveOutPath = os.path.join(cfgAva, 'Outputs/')
+    if not os.path.exists(saveOutPath):
+        # log.info('Creating output folder %s', saveOutPath)

```

Comment:
commented on purpose?

Reply:
I create the logger after this line so this is not printed anyway... I have to find a way to make it work.

Reply:
I think the log could end up in the root of the avalanche , so this would make it easier, right?

Reply:
Mh, i think the log file can end up in the avalanche root, so this would make things easier, right?